### PR TITLE
Update the homepage example server

### PR DESCRIPTION
**Pull request by johnbrett**
Updated the homepage example to use es6 and hapijs styleguide

### CODE REVIEW

File: templates/index.jade [hunk 1]
```diff
@@ -142,8 +153,16 @@ block content
           span.pl-c // Start the server
           br
 
-          | server.start(function () {
-          |     console.log('Server running at:', server.info.uri);
+          | server.start(() => {
```
Comment (AdriVanHoudt) [inline templates/index.jade]: server.start returns an error (if there is one), maybe also deal with that

File: templates/index.jade [hunk 2]
```diff
@@ -142,8 +153,22 @@ block content
...
+          | server.start((err) => {
+          br
+          
+          br
+          span.pl-s     if 
+          | (err) {
+          br
+          span.pl-s         throw 
+          | err;
+          |     }
```
Comment (johnbrett) [inline templates/index.jade]: @AdriVanHoudt that better? :)

Comment (AdriVanHoudt) [inline templates/index.jade]: that jade is not super readable but looks good :+1:

**Comment by nlf** [comment]
:+1: thanks!

**Pull request merged**